packages/components/src/components/hds/table/index.ts

```diff
@@ -4,29 +4,110 @@
  */
 
 import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { assert } from '@ember/debug';
+import { tracked } from '@glimmer/tracking';
+import type { ComponentLike } from '@glint/template';
+
+import type { HdsFormCheckboxBaseSignature } from '../form/checkbox/base';
+import type { HdsTableTdArgs } from './td.ts';
+import type { HdsTableThArgs } from './th.ts';
+import type { HdsTableThSortArgs } from './th-sort.ts';
+import type { HdsTableTrArgs } from './tr.ts';
+import {
+  HdsTableDensityValues,
+  HdsTableThSortOrderValues,
+  HdsTableVerticalAlignmentValues,
+} from './types.ts';
+import type {
+  HdsTableColumn,
+  HdsTableDensities,
+  HdsTableHorizontalAlignment,
+  HdsTableSelectableRow,
+  HdsTableSortingFunction,
+  HdsTableThSortOrder,
+  HdsTableVerticalAlignment,
+} from './types';
+
+export const DENSITIES: string[] = Object.values(HdsTableDensityValues);
```

Comment:
Could this instead be

`export const DENSITIES: HDsTableDensity[] = Object.values(HdsTableDensityValues);`

Reply:
Yes! 
```
export const DENSITIES: HdsTableDensities[] = Object.values(
  HdsTableDensityValues
);
```